### ISSUE TITLE
Differences between browser based Google Lighthouse performance & lighthouse-parade

### ISSUE THREAD

**Issue by mgifford** [cloudfour/lighthouse-parade#135]
I'm noticing a considerable difference between the score from Lighthouse Parade and from any of the pages in Google Lighthouse. I'm not sure why they would be different. It definitely decreases the credibility of the scan.  I'm just using something like:

`npx lighthouse-parade https://example.org/  ./lighthouse-parade-data --max-crawl-depth 3`

**Comment by calebeby** [comment]
@mgifford I am wondering if it might have to do with lighthouse-parade using a different browser version and/or lighthouse version than the lighthouse in your browser. I know that the lighthouse team changes how they record scores between versions.

`lighthouse-parade@2.1.0` uses lighthouse `9.5.0`
`lighthouse-parade@3.0.0-beta.2` uses lighthouse `9.6.2`. The latest lighthouse version is `10.1.1`, and I'll plan to update lighthouse before publishing the final version of lighthouse-parade@3.

I would guess that the version difference is probably a big factor in differences you might be seeing. If you're still seeing big differences after we get the lighthouse version in lighthouse-parade up to date, we can dig further into what might be causing differences.

**Comment by mgifford** [comment]
Good point.. And I might well want to ensure I'm upgrading to provide the latest of both libraries. They may be out of sync.

**Comment by mgifford** [comment]
I think I had installed this via NPM and was using the 2.x version of the code.  
  
I also don't see the instructions to build from the next branch of the repo [https://github.com/cloudfour/lighthouse-parade/tree/next](https://github.com/cloudfour/lighthouse-parade/tree/next)

How do I install the script for the 3.x version of the code?

**Comment by mgifford** [comment]
See https://github.com/cloudfour/lighthouse-parade/issues/71#issuecomment-1528936325
